src/dlgConnectionProfiles.h

```diff
@@ -44,6 +44,9 @@ class dlgConnectionProfiles : public QDialog, public Ui::connection_profiles
     QPair<bool, QString> writeProfileData(const QString& profile, const QString& item, const QString& what);
     QString readProfileData(const QString& profile, const QString& item) const;
     void accept() override;
+    QList<QListWidgetItem*> findData(const QListWidget& listWidget, const QVariant& what, const int role = Qt::UserRole) const;
+
+    const static int csmNameRole{Qt::UserRole};
```

Comment:
```suggestion
    static const int csmNameRole{Qt::UserRole};
```

Reply:
CodeFactor wants the `static` at the beginning of the type specification... :roll_eyes: